file-events/src/file-events/cpp/win_fsnotifier.cpp

```diff
@@ -84,15 +84,16 @@ bool WatchPoint::isValidDirectory() {
 }
 
 ListenResult WatchPoint::listen() {
-    BOOL success = ReadDirectoryChangesW(
+    BOOL success = ReadDirectoryChangesExW(
         directoryHandle,              // handle to directory
         &buffer[0],                   // read results buffer
```

Comment:
Do you think we need to increase the buffer size, now that the data structure is a little bit bigger?

Reply:
It's just a default, and I have no good idea about how to size it properly, so I'd leave it at the current value. We reconfigure it to 64K on the Gradle side anyway, because that's the maximum supported on network drives.